share/templates.d/99-generic/runtime-postinstall.tmpl

```diff
@@ -22,6 +22,13 @@ append etc/mdadm.conf "HOMEHOST <ignore>\n"
 remove etc/systemd/system/default.target
 symlink /lib/systemd/system/anaconda.target etc/systemd/system/default.target
 
+## Allow pipewire.service and pipewire.socket to run as root
```

Comment:
I think it would be cleaner if we used `install` here instead of hard-coding them in the script.

Reply:
install what? these files don't exist anywhere else. These are just little override snippets - the /usr/lib/systemd/user/pipewire.socket and pipewire.service in the pipewire package now have a line `ConditionUser=!root` that blocks them from running as root, these snippets override that.

I could put these snippets in a package and have lorax install them, but that seems like a lot of work...

Reply:
`share/templates.d/99-generic/config_files/common` has a bunch of config files we install with `install`, take a look at https://github.com/weldr/lorax/blob/master/share/templates.d/99-generic/runtime-postinstall.tmpl#L62

Reply:
ah, ok, cool, I'll look at doing it that way.